templates/components/cart/content.html

```diff
@@ -47,7 +47,15 @@ <h2 class="cart-item-name">
                                     {{#if is_file}}
                                         <a href="/viewfile.php?attributeId={{id}}&cartitem={{../id}}">{{value}}</a>
                                     {{else}}
-                                        {{{ sanitize value}}}
+                                        {{#if value '==' 'Yes'}}
```

Comment:
Can we use strict equality here?

Reply:
@bc-vlad-dlogush we can use strict equality and this will also work, but the stencil-object gives only the string "Yes" or "No" as the checkbox value, and I think that the additional check for the type of the resulting value can be omitted in this case.